app/components/post_list/post_list.test.js

```diff
@@ -86,4 +97,31 @@ describe('PostList', () => {
         instance.scrollToIndex(indexInRange);
         expect(flatListScrollToIndex).toHaveBeenCalled();
     });
+
+    test('should load more posts if available space on the screen', () => {
+        const wrapper = shallow(
+            <PostList {...baseProps}/>
+        );
+        const instance = wrapper.instance();
+        instance.loadToFillContent = jest.fn();
+
+        wrapper.setProps({
+            extraData: true,
+        });
+        expect(instance.props.extraData).toBe(true);
+
+        wrapper.setState({
+            postListHeight: 500,
+            contentHeight: 200,
+        });
+        expect(instance.state.postListHeight).toBe(500);
+        expect(instance.state.contentHeight).toBe(200);
+
+        wrapper.setProps({
+            extraData: false,
+        });
+
+        expect(instance.props.extraData).toBe(false);
+        expect(instance.loadToFillContent).toHaveBeenCalledTimes(1);
```

Comment:
Test works fine but few comments
- assertions to state and props after setting those are not necessary since it's how the enzyme should work.
- last assertion of `toHaveBeenCalledTimes(1)` is a bit misleading 'cause it seems to trigger when setting `extraData` to false which is not true.
My suggestion would be:
```javascript
// should not load to fill content
wrapper.setProps({extraData: true});
expect(instance.loadToFillContent).toHaveBeenCalledTimes(0);

// should load to fill content
wrapper.setState({postListHeight: 500, contentHeight: 200});
expect(instance.loadToFillContent).toHaveBeenCalledTimes(1);

// should not load to fill content
wrapper.setProps({extraData: false});
expect(instance.loadToFillContent).toHaveBeenCalledTimes(1);
```

Reply:
sounds perfect!!! Thanks @saturninoabril